CDK is missing `faros-js-client` dependency

I just installed the CDK package:

```
"dependencies": {
    "faros-airbyte-cdk": "^0.12.4"
},
```

I added a simple AirbyteDestination implementation with a unit test and the test failed:

```
Error: Cannot find module 'faros-js-client'
Require stack:
- MY_REPO_PATH/node_modules/.pnpm/faros-airbyte-cdk@0.12.4/node_modules/faros-airbyte-cdk/lib/sources/streams/stream-base.js
 ❯ Object.<anonymous> ../../node_modules/.pnpm/faros-airbyte-cdk@0.12.4/node_modules/faros-airbyte-cdk/src/sources/streams/stream-base.ts:1:
```

I noticed that JS client is not part of the CDK dependencies or listed as a peer dependency.

--- a/faros-airbyte-cdk/package.json
+++ b/faros-airbyte-cdk/package.json
@@ -40,6 +40,7 @@
     "@apidevtools/json-schema-ref-parser": "^9.0.9",
     "commander": "^9.3.0",
     "enquirer": "^2.3.6",
+    "faros-js-client": "^0.5.2",
     "fast-redact": "^3.0.2",
     "json-schema-traverse": "^1.0.0",
     "lodash": "^4.17.21",

